src/lib.rs

```diff
@@ -24,6 +24,9 @@ mod util;
 #[cfg(feature = "simple_http")]
 pub mod simple_http;
 
+#[cfg(feature = "simple_minreq")]
+pub mod simple_minreq;
```

Comment:
In d20ba2eafcaa3cd5245f5a548cbcbb6d14c13432:

Module name should be `minreq_http`.